src/third-party/gutenberg.stories.mdx

```diff
@@ -0,0 +1,568 @@
+          <p>The wren<br>Earns his living<br>Noiselessly.</p>
+          <p>— Kobayashi Issa (一茶)</p>
+        </div>
+      </div>`}
+  </Story>
+</Preview>
+
+## Preformatted
+
+The preformatted block allows theming of content within `pre` tags using the `wp-block-preformatted`
+class.
+
+<Preview>
+  {/* eslint-disable no-useless-escape */}
+  <Story name="core/preformatted" height="350px">
+    {`<pre class="wp-block-preformatted">
+       ___________________________
+     < I'm an expert in my field. >
+       ---------------------------
+           \\  ^__^
+            \\ (oo)\\_______
+              (__)\\       )\\\/\\
+                  ||----w |
+                  ||     ||
+      </pre>`}
+  </Story>
+  {/* eslint-enable no-useless-escape */}
+</Preview>
+
+## Pullquote
+
+The pullquote block wraps a `blockquote` element in a `figure` element with the class
+`wp-block-pullquote`.
+
+[Color controls](#color-settings) allow the setting of pre-defined or custom `background-color` and
+text `color`. A styles control adds one of two wrapper classes, `is-style-default` or
+`is-style-solid-color`. By default, any user-selected `background-color` will be
+applied only if the solid color background is toggled on.
+
+<Preview>
+  <Story name="core/pullquote" height="200px">
+    {`<figure class="wp-block-pullquote has-background is-style-solid-color" style="background-color:#dcd7ca">
+        <blockquote class="has-text-color has-accent-color">
+          <p>One of the hardest things to do in technology is disrupt yourself.</p>
+          <cite>Matt Mullenweg</cite>
+        </blockquote>
+      </figure>`}
+  </Story>
+</Preview>
+
+## Quote
+
+The quote block adds class of `wp-block-quote` directly to a `blockquote` element.
+It has a control to add an optional `is-style-large` class.
+
+<Preview>
+  <Story name="core/quote" height="150px">
+    {`<blockquote class="wp-block-quote is-style-large">
+        <p>In quoting others, we cite ourselves.</p>
+        <cite>Julio Cortázar</cite>
+      </blockquote>`}
+  </Story>
+</Preview>
+
+## Separator
+
+The separator block adds the `wp-block-separator` class to an `hr` element.
+
+Style controls add `is-style-default`, `is-style-wide` or `is-style-dots` classes.
+
+The color controls, if used, can add both `has-text-color` and `has-background`
+classes, as well as modifier classes for predefined color choices.
+
+<Preview>
+  <Story name="core/separator" height="100px">
+    {`<hr class="wp-block-separator has-text-color has-background has-subtle-background-background-color has-subtle-background-color is-style-wide">`}
+  </Story>
+</Preview>
+
+## Spacer
+
+The spacer block is a `div` with a class of `wp-block-spacer`.
+
+<Preview>
+  <Story name="core/spacer" height="225px">
+    {`<p>Content before spacer.</p>
+        <div
+          style="height:50px"
+          aria-hidden="true"
+          class="wp-block-spacer"
+        ></div>
+      <p>Content after spacer.</p>`}
+  </Story>
```

Comment:
Weird... is this meant to control margins in our WYSIWYG editor? Hopefully we can avoid this and use our margin utilities.

Reply:
Yeah, this is simply a way to add a random amount of vertical space, but potentially with additional styling. And it's probably something that won't get used, but I don't know. In general, I don't know whether we might un-register some of the default blocks in our theme to prevent their use. TBD.